Formula/gnuplot.rb

```diff
@@ -27,9 +27,9 @@ class Gnuplot < Formula
   deprecated_option "with-x" => "with-x11"
   deprecated_option "pdf" => "with-pdflib-lite"
   deprecated_option "wx" => "with-wxmac"
-  deprecated_option "qt" => "with-qt@5.7"
-  deprecated_option "with-qt" => "with-qt@5.7"
-  deprecated_option "with-qt5" => "with-qt@5.7"
+  deprecated_option "qt" => "with-qt"
+  deprecated_option "with-qt5" => "with-qt"
+  deprecated_option "with-qt@5.7" => "with-qt"
```

Comment:
I think maybe we shouldn't do this deprecation since it was octave forcing it. Better to have people fallback to the bottle if it was the only option they were using.